builders/xcm/xcm-sdk/xcm-sdk.md

```diff
@@ -0,0 +1,849 @@
+|       `asset`        |                                                                                                                                                       the [asset](#assets) to be transferred                                                                                                                                                       |
+| `existentialDeposit` | the [existential deposit](https://support.polkadot.network/support/solutions/articles/65000168651-what-is-the-existential-deposit-#:~:text=On%20the%20Polkadot%20network%2C%20an,the%20Existential%20Deposit%20(ED).){target=_blank}, or the minimum amount an address must <br> hold to be considered active if one exists, otherwise `0n` is returned |
+|        `min`         |                                                                                                                                                          the minimum transferrable amount                                                                                                                                                          |
+|       `native`       |                                                                                                                                                  the native [asset](#assets) of the source chain                                                                                                                                                   |
+|       `origin`       |                                                                                                                                the chain information for where the asset being transferred natively originates from                                                                                                                                |
+|       `source`       |                                                                                                                                      the chain information for where the asset being transferred is sent from                                                                                                                                      |
+|   `sourceBalance`    |                                                                                                                                           the balance of the asset being transferred on the source chain                                                                                                                                           |
+|  `sourceFeeBalance`  |                                                                                                                                            the fee for the asset to be transferred on the source chain                                                                                                                                             |
```

Comment:
This is not the fee itself but the balance to pay a fee. This will be present e.g. when you transfer `KBTC` because then you pay the fee with `KINT`. Otherwise, this field is `undefined`.